internal/verify/verify_test.go

```diff
@@ -42,6 +42,12 @@ type TestCase struct {
 	want    *policyv1.TestResults
 }
 
+func TestReloadGolden(t *testing.T) {
+	testCases := test.LoadTestCases(t, filepath.Join("verify", "cases"))
+	eng := mkEngine(t)
+	updateGoldenFiles(t, eng, testCases)
+}
```

Comment:
I don't think this is needed? Just run `go test -v -tags=tests ./internal/verify/... --args -updateGolden` to update the golden files.